.agent/hooks/test_hooks.py

```
@@ -0,0 +1,277 @@
+        )
+        monkeypatch.setattr("session_end._get_config_path", lambda: config_path)
+        import session_end
+
+        with mock.patch.object(
+            session_end, "_save_session_memory", return_value=True
+        ) as mock_save:
+            result = _run_hook(
+                session_end,
+                {
+                    "session_id": "s2",
+                    "transcript_path": "/tmp/transcript.json",
```

Ruff (0.15.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/transcript.json"

(S108)